CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changes Since v2.6.0
+
+  * Singularity is now written primarily in Go to bring better integratiion with the existing container ecosystem
+  * Added support for new URIs (`build` & `run/exec/shell/start`):
+    * `library://` - Supports the [Sylabs.io Cloud Library](https://cloud.sylabs.io/library)
+    * `docker-daemon:` - Supports images managed by the locally running docker daemon
+    * `docker-archive:` - Supports archived docker images
+    * `oci:` - Supports oci images
+    * `oci-archive:` - Supports archived oci images
+  * Handling of `docker` & `oci` URIs/images now utilizes [containers/image](https://github.com/containers/image) to parse and convert those image types in a supported way
+  * Replaced `singularity instance.*` command group with `singularity instance *`
+  * The command `singularity help` now only provides help regarding the usage of the `singularity` command. To display an image's `help` message, use `singularity run-help <image path>` instead
+ 
+## Removed Deprecated Commands
```

Comment:
Three hashes instead of two on these headers for consistency with the style we've previously adopted.